### ISSUE TITLE
ParticleNet for v14 geometry

### ISSUE THREAD

**Issue by LiamBrennan-UCSB** [IncandelaLab/LDMX-scripts#26]
## ParticleNet Paper

https://arxiv.org/pdf/1902.08570.pdf

This can be quite daunting to start. Try to read and familiarize yourself with the Network Architecture so that when you see these functions in the code you can tie them back mentally.

## How to Setup ParticleNet

1. Follow the Create POD account, tutorial: https://github.com/IncandelaLab/LDMX-scripts/wiki

2. Install ROOT on local: https://root.cern/install/

- There's tons of ROOT commands and a good ROOT wiki as well as some tutorials: https://root.cern.ch/root/htmldoc/guides/primer/ROOTPrimer.html 

3. Install ParticleNet on POD, tutorial: https://github.com/IncandelaLab/LDMX-scripts/tree/master/GraphNet

4: Transfer a single file from POD to a local directory

- scp works by: scp (file to transfer) (where to transfer to)
- scp: "scp (your_account)@pod-login1.cnsi.ucsb.edu://home/aminali/production/v14_prod/v3.2.0_ecalPN_tskim_sizeskim/mc_v14-4.0GeV-1e-ecal_photonuclear_batch9_runs44_sizeskim.root (location on your computer)" 

- Run ROOT with: root --web=off
- open a TBrowser with the command: new TBrowser()
- Find the file you just transferred over and look at some of the leafs to familiarize yourself with the naming conventions and data

5. Move a pre-processing script to your directory on POD if it isn't already there: /home/lbrennan/LDMX-scripts/GraphNet/file_processor.py

6. Change the file_templates from V13 to V14 and run the pre-processing script 

- This code will not work immediately. It was written for V12. Use ROOT to figure out what to changes are needed to get this working for V14 samples.

There are some other key differences between V12 and V14. Once you've completed these steps we can discuss the other variables that will need to be changed.

## Existing material to start off

V14 Samples
/home/aminali/production/v14_prod
Background: v3.2.0_ecalPN_tskim_sizeskim
Signal: Ap0.001GeV_1e_v3.2.2_v14_tskim, Ap0.01GeV_1e_v3.2.2_v14_tskim, Ap0.1GeV_1e_v3.2.2_v14_tskim, Ap1GeV_1e_v3.2.2_v14_tskim

 * Read through the Dataset.py code and train.py code. Try to understand what is going on and why. Make some slides if that will help
 * Create V14 4Gev plots.
 * Afterwards begin to work on 8GeV plots / we can discuss where to go

**Comment by tvami** [comment]
> Install ParticleNet on POD, tutorial: https://github.com/p-masterson/LDMX-scripts/tree/lazy-loading/GraphNet

can we move Phillip's files to this main repo please? @LiamBrennan-UCSB

**Comment by LiamBrennan-UCSB** [comment]
Yes I can work on that

**Comment by tvami** [comment]
@DuncanWilmot Phillip's stuff is now in the main repo, right? @LiamBrennan-UCSB please modify the description to reflect that.

**Comment by DuncanWilmot** [comment]
@tvami @LiamBrennan-UCSB yes, v13 stable version of ParticleNet located at https://github.com/IncandelaLab/LDMX-scripts/tree/master/GraphNet

**Comment by tvami** [comment]
> please modify the description to reflect that.

nvm, I did this
